sdc/functions/numpy_like.py

```diff
@@ -201,6 +201,20 @@ def sdc_isnan_float_impl(self):
     ty_checker.raise_exc(dtype, 'int or float', 'self.dtype')
 
 
+def gen_sum_bool_impl():
+    """Generate sum bool implementation."""
+    def _sum_bool_impl(self):
+        length = len(self)
+        result = 0
+        for i in prange(length):
+            if self[i]:
```

Comment:
Why do you need this `if self[i]` condition?

Reply:
Without it also works, fixed